tdrs-backend/docker-compose.yml

```diff
@@ -95,7 +95,6 @@ services:
       bash -c "./wait_for_services.sh &&
                ./gunicorn_start.sh && celery -A tdpservice.settings worker -l info"
     ports:
-      - "8080:8080"
       - "5555:5555"
```

Comment:
why does the port change? looking through the rest of the configs, i see that gunicorn is still started at `8080`, but i don't see a mapping to that.

Reply:
We do not need the mapping anymore. The external-net takes care of the mapping and can connect to gunicorn using the actual ip+port of instance. That way, we could use the port 8080 on the Nginx in frontend.